src/App.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import "./styles/global.css";
+import Drum from "./components/Drum";
+import drumKits from "./data/drumKits";
+
+function App() {
+  const [selectedKitName, setSelectedKitName] = useState(drumKits.find((kit) => kit.kitName === 'Original'));
+  const handleKitChange = (e) => {
+    setSelectedKitName(e.target.value);
```

Comment:
please make it consistent when you are setting the state, are you setting the kitName string value, or the kit object